src/lib/rest/rest.cpp

```diff
@@ -1025,7 +1025,7 @@ static int connectionTreat
   // 3. Finally, serve the request (unless an error has occurred)
   if (((ciP->verb == POST) || (ciP->verb == PUT)) && (ciP->httpHeaders.contentLength == 0) && (strncasecmp(ciP->url.c_str(), "/log/", 5) != 0))
   {
-    std::string errorMsg = restErrorReplyGet(ciP, ciP->outFormat, "", url, SccLengthRequired, "Zero/No Content-Length in PUT/POST request");
+    std::string errorMsg = restErrorReplyGet(ciP, ciP->outFormat, "", url, SccLengthRequired, "Zero/No Content-Length in PUT/POST/PATCH request");
```

Comment:
Have you run the functional tests (i.e. `make ft`)  to check that all the existing tests are ok? Typically, when output changes, some test need litle adjusts.